Azure Firewall/Workbook - Azure Firewall Monitor Workbook/Azure Firewall_ARM.json

```diff
@@ -10,35 +10,16 @@
     },
     "workbookType": {
       "type": "string",
-      "allowedValues": [
-          "workbook",
-          "sentinel"
-          
-        ],
-        "defaultValue": "workbook",
+      "defaultValue": "workbook",
       "metadata": {
         "description": "The gallery that the workbook will been shown under. Supported values include workbook, tsg, etc. Usually, this is 'workbook'"
       }
     },
-	"DiagnosticsWorkspaceName": {
+    "workbookSourceId": {
       "type": "string",
-      "defaultValue": "<WorkspaceName>",
+      "defaultValue": "Azure Monitor",
       "metadata": {
-        "description": "Provide the workspace name for your Network Diagnostic logs"
-      }
-    },
-    "DiagnosticsWorkspaceSubscription": {
-      "type": "string",
-      "defaultValue": "<WorkspaceSubscriptionID>",
-      "metadata": {
-        "description": "Provide the workspace subscription GUID for your Network Diagnostic logs"
-      }
-    },
-    "DiagnosticsWorkspaceResourceGroup": {
-      "type": "string",
-      "defaultValue": "<ResourceGroupName>",
-      "metadata": {
-        "description": "Provide the workspace resourcegroupname for your Network Diagnostic logs"
+        "description": "The id of resource instance to which the workbook will be associated"
```

Comment:
This field used to be the name of the resource group where the Log Analytics workspace reside. 
This proposed update is asking for the ID of the resource for which it would be associated, in this case An Azure Firewall. 
How will a customer call an existing Log analytics workspace in this case? And what happens when the resource is deleted?

<img width="411" alt="resourceID" src="https://user-images.githubusercontent.com/5616715/199561843-899219b2-bc71-4b76-9491-a96fb2e21c98.png">